test/dynamo/test_export.py

```diff
@@ -3637,34 +3637,40 @@ def forward(self, pred, x):
     ones_3 = torch.ones(6, 4)
     cond_true_0 = self.cond_true_0
     cond_false_0 = self.cond_false_0
-    cond = torch.ops.higher_order.cond(arg0, cond_true_0, cond_false_0, [arg1, ones, ones_1, ones_3, ones, ones_1, ones_2]);  arg0 = cond_true_0 = cond_false_0 = arg1 = ones = ones_1 = ones_3 = ones_2 = None
+    cond = torch.ops.higher_order.cond(arg0, cond_true_0, cond_false_0, [ones, ones_1, arg1, ones_3, ones_2]);  arg0 = cond_true_0 = cond_false_0 = ones = ones_1 = arg1 = ones_3 = ones_2 = None
```

Comment:
cc @voznesenskym why don't these variable names get named better? (e.g. ones_1, ones_2, ones_3) would ideally be (a, b, c)

Reply:
If you run with `TORCH_LOGS=+dynamo` and breakpoint in `lazy_format_graph_code`, called from `compile_and_call_fx_graph`, the graph looks right:

```
def forward(self, L_pred_ : torch.Tensor, L_x_ : torch.Tensor):
    l_pred_ = L_pred_
    l_x_ = L_x_
    a = torch.ones(6, 4)
    b = torch.ones(6, 4)
    c = torch.ones(6, 4)
    d = torch.ones(6, 4)
    cond_true_0 = self.cond_true_0
    cond_false_0 = self.cond_false_0
    cond = torch.ops.higher_order.cond(l_pred_, cond_true_0, cond_false_0, [a, b, l_x_, d, c]);  l_pred_ = cond_true_0 = cond_false_0 = l_x_ = None
    return (d, c, b, a, cond)
```

Setting a breakpoint in the export interceding compiler also shows the gm as correctly printing.

Printing after rewrite_signature shows it as changing to a worse version. I can fix this rq.

Reply:
https://github.com/pytorch/pytorch/pull/111947